### ISSUE TITLE
Publishing the FROST chart on Helm Hub

### ISSUE THREAD

**Issue by rouvoy** [FraunhoferIOSB/FROST-Server#90]
Did you consider publishing an official FROST server on [Helm Hub](https://hub.helm.sh/)?

**Comment by phertweck** [comment]
Yes, we thought about publishing to Helm Hub, but haven't found the time, yet. To do so we need to check the formal requirements, first.
